Exercices/JS06/index.js

```diff
@@ -0,0 +1,11 @@
+"use strict"; 
+alert("Enter the current hour");
+let hour = prompt("insert hour between 0 - 23:");
+Number(hour);
```

Comment:
It's good that you used `Number(hour)` conversion here.
But Number() works in away that it does conversion and then returns newly converted value.
if you don't assign it to anything, you conversion happened but you didn't store it anywhere.
Similar would be to use just `prompt()`.
Think about it and do improvements.
Tip: Variable can be reassigned,